src/ripple/shamap/impl/SHAMapDelta.cpp

```diff
@@ -143,11 +143,11 @@ bool SHAMap::compare (SHAMap::ref otherMap, Delta& differences, int maxCount)
         if (ourNode->isLeaf () && otherNode->isLeaf ())
         {
             // two leaves
-            if (ourNode->getTag () == otherNode->getTag ())
+            if (ourNode->peekItem()->getTag () == otherNode->peekItem()->getTag ())
```

Comment:
More like "look at a reference" as opposed to "make a copy" (which is usually indicated by "get").